### ISSUE TITLE
`Error: conn closed` after cancelling a query

### ISSUE THREAD

**Issue by johnsmyth** [turbot/steampipe#3791]
**Describe the bug**
sometimes after I CTRL-C to cancel a query, I get `conn closed` on subsequent queries:
```
> select jsonb_pretty(_ctx),topic_arn from aws_sns_topic

Error: execution cancelled

... results here ....

> select jsonb_pretty(_ctx),* from aws_sns_topic
Error: conn closed
```

Once I receive this error, I need to restart steampipe to clear it.  On some (but not all) occasions, steampipe has not shut down cleanly and i need to `pkill` it

**Steampipe version (`steampipe -v`)**
Steampipe v0.21.0-alpha.6

**To reproduce**
1. Start a long running query
2. cancel it with ctrl-c
3. try to run another query

**Comment by binaek** [comment]
Thank you for reporting this @johnsmyth  . We will get a fix out asap

**Comment by binaek** [comment]
Should be fixed by #3784
